Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+
+Besides scanning seeds, you can also use this view to scan SSKR shares (See "Importing SSKR Shares") or respond to a `ur:crypto-request` (see "Answering Requests"), as described below.
+
+### Importing a Seed via Cut and Paste
+
+The **add** ("+") button below the main menu gives a number of options for creating seeds in **Seed Tool**, but also lets you input text as hex bytes, `ur:crypto-seed`, Bytewords, or BIP39 Mnemonic Words. In each case, you just type or paste the words, and then click "Done".
+
+The following show examples of the data you might input for each data type:
+
+* **Hex Bytes:** 59F2293A5BCE7D4DE59E71B4207AC5D2
+* **`ur:crypto-seed`:** `ur:crypto-seed/oyadgdhkwzdtfthptokigtvwnnjsqzcxknsktdhpyljeda`
+* **Byte Words:** hawk whiz diet fact help taco kiwi gift view noon jugs quiz crux kiln silk tied omit keno lung jade
+* **BIP 39 Mnemonic Words:** fly mule excess resource treat plunge nose soda reflect adult ramp planet
+
+Again, you can also add add SSKR shares, as described below.
+
+### Importing SSKR Shares
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-import.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-sskr-import.jpeg" align="right" width=250></a>
+
+SSKR stands for Sharded Secret Key Reconstruction, a Blockchain Commons [specification](https://github.com/BlockchainCommons/Research/blob/master/papers/bcr-2020-011-sskr.md). It allows you to shard a secret (such as a seed) into a number of shares, and then reconstruct the secret from some number (threshold) of those seeds that's typically fewer than all of them. For example, you might shard a seed into three shares with a threshold of two. Typically, the shares are given out to friends and family, placed in distant safety deposit boxes, or otherwise separated. A threshold of the shares can then be collected and used to reconstruct the seed if the original is lost. 
+
+**Gordian Seed Tool** can be used to collect together shares and reconstruct your seed. There are currently three ways to do so:
+
+* **Scan: QRs.** Photograph QRs of SSKR shares until you have a threshold.
+* **Scan: Paste Crypto-SSKR.** Paste `ur:crypto-sskr` of SSKR shares until you have a threshold.
+* **Add: Shares.** Simultaneouly paste sufficient shares to meet a threshold into the box.
+
+The `ur:crypto-sskr` specification, which can be used in the `Add` section, in the `Scan: Paste`, or encoded as a QR, look like this:
+
+* ur:crypto-sskr/gobnbdaeadaevdbkclhseeehtldedikgpysoenreceeeeorofrwn
+* ur:crypto-sskr/gobnbdaeadadrkhesefhjzdycypduokkjejponsayabguymwwnwz
+
+The SSKR words, which can only be used in the `Add` section, look like this:
+
+* tuna acid epic gyro gray tent able acid able frog cusp heat poem owls undo holy rich monk zaps cost brag loud fair dice idle skew iris beta tied
```

Comment:
Maybe want to explain somewhere that the first byte words are the same, and in some scenarios can be excluded (for instance Ken excludes them). It is the last 4 words that uniquely identifies them, which is why they are highlighted in bold on the printed version.

Reply:
I would *really* like to avoid promoting the idea of dropping ByteWord to save space. It's an esoteric use for advanced users. Every ByteWord has a purpose and I don't think we should be suggesting to average users that some of them might not be necessary.

Reply:
Going to punt this, and say we don't need to get into this complexity, and we don't want to encourage saving space. I think it was relevant for Ken's example of scribing onto dog tags, but it's rarely desired elsewhere because it costs us our identification (at the front) or checksums (at the back).